back-end/java/main/java/src/DormRoom/DormBuilding.java

```diff
@@ -0,0 +1,192 @@
+package src.DormRoom;
+
+/**
+ * The {@code DormBuilding} class represents data about a specific dormitory building on campus.
+ * Each instance contains information such as the building's name, its location (latitude
+ * and longitude), the year it was built, and its corresponding campus area. Additionally,
+ * it captures the ratio of the building's occupancy to the number of available washing machines
+ * (peoplePerWasher), which provides insight into the building's laundry capacity.
+ *
+ * <p>Constructors create an object fully initialized with the building's attributes,
+ * inferred from the building name. This approach ensures that instances of {@code DormBuilding}
+ * remain consistent and standardized across the application.</p>
+ */
+public class DormBuilding {
```

Comment:
I like the use of the final keywords here. You should add public / private to the fields as well. Probably public since we want to access this data across all classes since everything is just a constant.